packages/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { FluentIconsProps } from '@fluentui/react-icons';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as ReactIcons from '@fluentui/react-icons';
+import { Input } from '@fluentui/react-input';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useStyles = makeStyles({
+  grid: theme => ({
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: ' flex-start',
+
+    '> span': {
+      alignItems: 'center',
+      color: '#3b3a39',
+      display: 'flex',
+      flexDirection: 'column',
+      height: '80px',
+      justifyContent: 'space-around',
+      padding: '8px',
+      width: '80px',
+      overflow: 'hidden',
+
+      '> div': {
+        fontSize: '11px',
+        opacity: '0',
+      },
+
+      '&:hover': {
+        overflow: 'visible',
+
+        '& div': {
+          opacity: '1',
+        },
+      },
+    },
+  }),
+
+  searchBox: {
+    maxWidth: '320px',
+  },
+
+  radio: {
+    fontSize: '11px',
+  },
+});
+
+const reactIcons: React.FC<ReactIcons.FluentIconsProps>[] = Object.keys(ReactIcons)
+  .map(iconName => ReactIcons[iconName])
```

Comment:
To address the build issue
```suggestion
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  .map(iconName => (ReactIcons as any)[iconName])
```